code/datums/shuttles.dm

```diff
@@ -219,6 +219,13 @@
 	admin_notes = "This shuttle will likely crush escape, killing anyone there."
 	credit_cost = -5000
 
+/datum/map_template/shuttle/emergency/stormrider
+	suffix = "stormrider"
+	name = "Asteroid With Engines Strapped To It"
+	description = "A cramped, reinforced shuttle with PDS installed at key points and absolutely no windows. This shuttle was designed for extreme exfiltration through dense debris fields by small teams of well-trained EVA operatives. Comes with a sleeper, a big ass engine, two lockers with internals and access hatches to make sure the asteroid defense systems didn't mysteriously not get activated when the shuttle departed Central Command."
+	admin_notes = "An antag-bombers wet dream. But at least it stops meteors..."
```

Comment:
It's bigger and sturdier than the  "Oh Hi Daniel", calling it an "antag-bomber wet dream" may be misleading, It's just a bit stripped down of furniture.